.travis.yml

```diff
@@ -0,0 +1,13 @@
+services:
+    - docker
+
+env:
+    - secure: "jq0r7xvk+XAETGrVRZ3gWgxVlkYTJs+W7WsYHu2VLx6JAweitL4MGgXn45mzgQGw+P4nA/KDZWw0VsvEluvA22kjsq87QtawMm3ZzqIUPe5U7wtdLIvLDSFcOb4SBEh6FfN2uLfLsECv4x4+2LyCZPLHmMuXOpFLd/xatC7wMBhEXt2YPdC2PYZnZ7UQDCNe9OJXzZrRUfLnHuUhBMGtd7hoCWjr84ePeBWg/PC6ec+4p+zajU9DQEzne0e44z7r//qU6O4dfLixke0jgPVBXdVEfSeM2UWL6tCFx+ntEqDR1w/c9qp3lfDEqcye2P6X+2BEqWbL5xv3bySnXvSDm8d3KEoy/XDef7CyfmImcMQGtMi9H1ZPBZ988rJVmJu8jLlrKIsx1VE5+ZekCuT9Ol0atiBWmoyyDd1y8Kumd9UKMicY7h7EeOJtV3b5fvLm1mK3/AEapqjpRrbTa4dKFQyz/7qZp+hJrf2Gf3YlAU8fOZeCDuamrU/3sBbR+vfI5zhErTp74PwgDJwmRAPWsMRb+aQjanD2VjPFND65FjwByvcQ3IOV4NSP3nq4d/Wu95Fzx59JSauTihR3716POaWq5/dzG6IXwdN30le39NJsI8zvl9UFGD0h7VwI2UeKODlLjGrgJ20UhiOJ3GdWmk1/E3B7iuGdJBrGE5kG0s4="
+before_install:
+    - docker build --target test --tag todo-app:test .
+
+script:
+    - docker run --mount type=bind,source="$(pwd)",target=/app todo-app:test tests/unit
+    - docker run --mount type=bind,source="$(pwd)",target=/app todo-app:test tests/integration
+    - docker run -e TRELLO_API_KEY=$trello_key -e TRELLO_API_SECRET=$trello_secret  --mount type=bind,source="$(pwd)",target=/app todo-app:test tests/end_to_end
```

Comment:
Environment variables are conventionally in all caps, so I expected `TRELLO_API_KEY=$TRELLO_KEY`

Also, if you make the env var names match, you can write this slightly more succinctly with `docker run -e TRELLO_API_KEY -e TRELLO_API_SECRET --mount ...etc`